src/client/lazy-app/Modal/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { h, Component, VNode, Fragment } from 'preact';
+import * as style from './style.css';
+import 'add-css:./style.css';
+import { linkRef } from 'shared/prerendered-app/util';
+import { cleanSet } from '../util/clean-modify';
+import { animateFrom, animateTo } from '../util';
+import { RoundedCrossIcon } from '../icons';
+
+interface Props {
+  icon: VNode;
+  title: string;
+  content: VNode;
```

Comment:
I think we decided that the content of the model should be the children of the modal?